lib/event.rb

```diff
@@ -0,0 +1,41 @@
+class Event
+  attr_reader :name,
+              :crafts,
+              :attendees
+  def initialize(name, crafts, attendees)
+    @name = name
+    @crafts = []
+    @attendees = []
+  end
+
+  def add_crafts(craft)
+    @crafts << craft
+  end
+
+  def add_attendees(person)
+    @attendees << person
+  end
+
+  def attendee_names
+    @attendees.map do |attendee|
+      attendee.name
+    end
+  end
+
+  def craft_with_most_supplies
+    craft = @crafts.max_by do |craft|
```

Comment:
Think about your variable naming here. What are you actually returning? I used `most_supplies`. This also did not pass the spec harness because of that? Nice use of `.max_by`.
Here is what I came up with
```
def craft_with_most_supplies
    most_supplies = @crafts.max_by do |craft|
      craft.supplies_required.length
    end
    most_supplies.name
  end
```